ExtractorProject/Settings/IgraslovProviderSettings.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Core.Settings;
+
+namespace ExtractorProject.Settings;
+
+/// <summary>
+/// настройки  для провайдера игры слов
+/// </summary>
+public class IgraSlovProviderSettings : IProviderSettingsInfo
```

Comment:
```suggestion
public class IgraSlovProviderSettingsInfo : IProviderSettingsInfo
```
Для лабиринта я потом тоже такое поменяю